packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -12,32 +12,20 @@ import {
   GEL_GROUP_4_SCREEN_WIDTH_MIN,
 } from '@bbc/gel-foundations/breakpoints';
 
-export const StoryPromoLi = styled.li.attrs({
+export const StoryPromoLiThin = styled.li.attrs({
   role: 'listitem',
 })`
-
-  @media (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-    border-bottom: 0.0625rem solid ${C_LUNAR};
-  }
-
   ${({ border }) =>
     !border &&
     `
-      @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
-          border-bottom: none;
-      }
-    `}
```

Comment:
It would make sense if this _added_ border styles instead of removed/overwrote them. If we want border to be the default, we can make use of `defaultProps`.

N.B. Currently `border` styles exist on this component by default, which means we have to add the inverse to remove them. This is made even more complex by the media query.

Reply:
I thought of that actually and was initially confused as to why it did that maybe that was a requirement when it was first built. The component is already used in a bunch of places so reversing that would mean we would need to go update all instances where it's been used.

Reply:
Yeah, it is confusing. I think there's an opportunity to fix this now. Here are the places it's being used in Simorgh: https://github.com/bbc/simorgh/search?q=psammead-story-promo-list&unscoped_q=psammead-story-promo-list.

Reply:
@simonsinclair let's do it 💪

Reply:
@simonsinclair done!